fred2/orienteditor.h

```diff
@@ -70,7 +71,10 @@ class orient_editor : public CDialog
 
 private:
 	float convert(const CString &str);
-	bool close(float val, const CString &str);
+	float perform_input_rounding(float val);
+
+	bool is_close(float val, const CString &input_str);
+	bool is_angle_close(float rad, const CString &input_str);
```

Comment:
Optional: maybe make these functions `const`, although as you know, `const` correctness can trigger a ripple effect, so maybe not worth it.

Reply:
Ok.